### DISCUSSION TITLE
.fr domain

### DISCUSSION THREAD

**Discussion by antonin-11** [vercel/vercel#8548] [Help]
I want to add a .fr domain name to Vercel but I get the following error message: "We do not support ".fr" TLD yet. So I'm wondering if there is any solution and if not when this feature will be available

**Comment by amyegan** [comment]
Hi @antonin-11! I believe the problem you're facing is that Vercel does not support purchasing .fr domains from the Dashboard at this time. You will still be able to use a custom .fr domain with your Vercel Project by purchasing it from an external domain name registrar and using A/CNAME records to assign it to your website. You can find a walkthrough at [vercel.com/docs/concepts/projects/custom-domains](https://vercel.com/docs/concepts/projects/custom-domains#step-1:-selecting-your-project).
